CHANGELOG.rst

```diff
@@ -3,6 +3,11 @@ UNRELEASED
 
 * Add new unit: "Stokes" (``St``).
 * Use ``TypeCheckingSupport`` from ``oop-ext 1.1``.
+* Add ``cubic feet per day per psi`` (``ft3/psi.d``) unit to ``productivity index`` category.
+* Add ``calories/metre hour degree Celsius`` (``cal/m.h.degC``) unit to ``thermal conductivity`` category.
+* Add ``calorie/hour square metre deg C`` (``cal/h.m2.degC``) unit to ``heat transfer coefficient`` category.
+* Add ``std cubic metres/second`` (``sm3/s``) unit to ``standard volume per time`` category.
+* Add ``megastd cubic feet/stock tank barrel`` (``MMscf/stb``), ``stock tank barrel/std cubic feet`` (``stb/scf``) and ``stock tank barrel/million std cubic feet`` (``stb/MMscf``) units to ``standard volume per standard volume`` category.
```

Comment:
Since you called ```stock tank barrel/million std cubic feet``` here perhaps change

```
``megastd cubic feet/stock tank barrel``
```
to

```
``million std cubic feet/stock tank barrel``
```

as well

Reply:
Nice catch! Changed to `million std cubic feet/stock tank barrel`